webpack.config.js

```diff
@@ -203,6 +202,15 @@ function getWebpackConfig( { externalizeWordPressPackages = false } = {}, argv )
 					test: /\.html$/,
 					loader: 'html-loader',
 				},
+				{
+					test: /\.(svg)$/,
+					use: [
+						{
+							loader: 'file-loader',
+							options: { name: '[name].[ext]', outputPath: 'images/' },
+						},
+					],
+				},
```

Comment:
this still gives me hesitation. do we want all SVG imports to return a URL? I'm not sure we do. I wondered about whether or not it makes sense to keep the loader specified in `flagUrl()` since we might plausibly pull in SVG files as the actual SVG strings.

maybe it's not worth thinking about yet since we haven't done this yet in the app? is copying a reasonable default?

Reply:
It doesn't really make much difference at this point, IMO. I'd argue that similarly to using the SASS loader (and friends) to handle our emerging SASS `import`s, it's probably okay to specify the file loader for SVGs globally (i.e. in the webpack configs), and modify later if needed.

Specifying it directly in the `import` statement might help wrt the server side tho (no duplication of `file-loader` stanzas across client and server side webpack config).